### DISCUSSION TITLE
RangeInput - always show two decimal places

### DISCUSSION THREAD

**Discussion by Adsecularis** [ctreffe/alfred#149] [Alfred Q&A] [answered]
Hi, I wanna add a RangeInput into a dictator game so test subjects can set a slider from 0€ to 10€ to decide how much money they want to give to the other person. I set the steps to 0.10.
However Alfred doesn't display the excess Zeros, so it starts at 5 € and goes to 5,1 €, 5,2 € and so on. But I would like it to be 5,00 € then 5,10 € and so on, so that it always shows two decimal places even when they are Zeros.
I just don't know how to do it.

The code is pretty much the example from the documentation with the parameters needed for the experiment.
```
import alfred3 as al
exp = al.Experiment()

@exp.member
class Demo(al.Page):

    def on_exp_access(self):
        self += al.RangeInput(
            name="range_demo",
            force_input=True,
            min=0.00,
            max=10.00,
            step=0.10,
            display_locale="de-DE",
            display_suffix=" €",
            display_input=True
        )


if __name__ == "__main__":
    exp.run()
```

**Comment by jobrachem** [comment, marked as answer]
The `RangeInput` element currently does not offer a straigthforward option for this - but I think it should. I'll create an issue from this question and expect that the feature will be available in the next minor update.

## Why does `step=0.10` not work?
It is a creative idea to include the trailing zero in the definition of the *step* argument - but Python does not recognize a difference between `0.1` and `0.10`

```python
>>> a = 0.1
>>> b = 0.10
>>> a == b
True
```
